### DISCUSSION TITLE
How to get the xaml result without writing a file.

### DISCUSSION THREAD

**Discussion by MrInventor** [ElinamLLC/SharpVectors#202] [Q&A] [answered]
I´m trying to convert svg to xaml using azure functions. 
The svg file is in a blop store and I would like to save the xaml result in another store at azure.

Currently:
_myBlop_ is a Stream that the function receives when a new file is uploaded. 

```
WpfDrawingSettings settings = new WpfDrawingSettings();
settings.IncludeRuntime = false;
settings.TextAsGeometry = true;
settings.EnsureViewboxPosition = true;

FileSvgConverter converter = new FileSvgConverter(false, false, settings);                 

converter.Convert(myBlob, Path.GetFileNameWithoutExtension(name));
```

How do I read the xaml result when saveXaml and saveZaml are set to false?

**Comment by paulushub** [comment, marked as answer]
@MrInventor There are several converters to handle different situations, all deriving from `SvgConverter`, which you can also extend to create a customized conversions. Here are some...
**StreamSvgConverter** - will convert and save static image to stream 
**FileSvgConverter** - will convert and same XAML to file
**FileSvgReader** - will convert to **DrawingGroup** and optionally save to file.

**FileSvgReader** is the most generic converter, so use it to convert to **DrawingGroup** and save that to the form you want in XAML using either the **XmlXamlWriter** or the framework **XamlWriter**. It is the same process used by the **FileSvgReader** to save to file.

**Reply by MrInventor** [reply]
Thanks for pointing me in the right direction 👍
